src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_IIfStatement.cs

```diff
@@ -0,0 +1,336 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.CSharp.Test.Utilities;
+using Roslyn.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.CSharp.UnitTests
+{
+    public partial class IOperationTests : SemanticModelTestBase
```

Comment:
>IOperationTests [](start = 25, length = 15)

Consider also testing scenarios:
- with embedded statements that are not blocks;
- with conditions declaring variables (out or pattern);
- with embedded statements declaring variables (out or pattern).